src/controllers/mouseoverTextController.js

```diff
@@ -1,43 +1,48 @@
-const mouseoverTextController = (function (MouseoverText) {
-    const createMouseoverText = function (req, res) {
-        const newMouseoverText = new MouseoverText();
-        newMouseoverText.mouseoverText = req.body.newMouseoverText;
-        newMouseoverText.save().then(() => res.status(201).json({
-            _serverMessage: 'MouseoverText succesfuly created!',
-            mouseoverText: newMouseoverText,
-        })).catch(err => res.status(500).send({ err }));
-    };
+const mouseoverTextController = function (MouseoverText) {
+  const createMouseoverText = function (req, res) {
+    const newMouseoverText = new MouseoverText();
+    newMouseoverText.mouseoverText = req.body.newMouseoverText;
+    newMouseoverText
+      .save()
+      .then((result) =>
+        res
+          .status(201)
```

Comment:
I agree that it's a convention to send the object created or updated for Update and Create operations (that's the standard), however this was not in the code before.

Is this a bugfix required? I do like the fact that you went ahead and updated the code to be correct (from a coding convention standpoint). Anyway, I don't think this will break anything. Remember that updating the source code can lead to frontend unexpected errors. Nicely done here though.